stellar-contract-env-host/src/host.rs

```diff
@@ -1122,7 +1122,7 @@ impl CheckedEnv for Host {
 
     fn map_keys(&self, m: Object) -> Result<Object, HostError> {
         self.visit_obj(m, |hm: &HostMap| {
-            let mut vec = self.vec_new()?;
+            let mut vec = self.vec_new(RawVal::from_void())?;
```

Comment:
The host knows the size of the map and should use that.

Reply:
Fixed.